src/lifx.d.ts

```diff
@@ -133,6 +143,14 @@ export class Client {
 
 export class Light {
 
+  public client : Client;
+  public id : string;
+  public address : string;
+  public port : number;
+  public label : string;
+  public status : string;
+  public seenOnDiscovery : number;
```

Comment:
I would not say these are explicitly public. Make these as `readonly` with the default visibility:
```ts
  readonly client: Client;
  readonly id: string;
  readonly address: string;
  readonly port: number;
  readonly label: string;
  readonly status: string;
  readonly seenOnDiscovery: number;
```